src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -202,6 +202,8 @@ public function handle(AddProductToOrderCommand $command)
             $this->orderAmountUpdater->update($order, $cart, (int) $orderDetail->id_order_invoice);
 
             $order->update();
+
+            $specificPrice->delete();
```

Comment:
Mmm, the purpose of this PR is to never delete the specific price any more They must stay in database and be updated as long as the price is different from catalog
The only case where you delete them is if the price is changed to match the catalog one